integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -276,8 +268,8 @@ def get_address_from_coordinates(request, region_slug):
 
     return JsonResponse(
         data={
-            "number": address.get("house_number") or "",
-            "street": address.get("road") or "",
+            "number": address.get("house_number"),
+            "street": address.get("road"),
```

Comment:
I think `None` is more meaningful than `""` here (and we're not showing these values if they're empty anyway)